engine/Factories.js

```diff
@@ -13,3 +13,12 @@ const createShader = function(gl, id) {
   console.error(gl.getShaderInfoLog(shader));
   gl.deleteShader(shader);
 };
+
+const meshFactory = function(gl, program) {
```

Comment:
I don't really get what the advantage of a factory is… Why can't a `Cuboid` just be a subclass of `Mesh` with a different constructor? Or why is there a Cuboid class and not a cuboid factory?

Reply:
Because I would like to separate GL context from the business objects.